openpgp/packet/signature_test.go

```diff
@@ -76,4 +76,57 @@ func TestSignUserId(t *testing.T) {
 	}
 }
 
+func TestSignatureWithPolicyURI(t *testing.T) {
+	testPolicy := []byte("This is a test policy")
+	sig := &Signature{
+		SigType:    SigTypeGenericCert,
+		PubKeyAlgo: PubKeyAlgoRSA,
+		Hash:       0, // invalid hash function
```

Comment:
Same thing here as in https://github.com/ProtonMail/go-crypto/pull/83#discussion_r662489631 :slightly_smiling_face: